src/common/PersonCard/styled.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+import { ReactComponent as noPicture } from "./noPicture.svg";
+
+export const Card = styled.div`
+    padding: 16px;
+    width: 208px;
+    background-color: ${({ theme }) => theme.colors.white};
+    box-shadow: ${({ theme }) => theme.boxShadow.tile};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media (max-width:${({ theme }) => theme.breakpoints.mobileMax}px) {
+        max-width: 136px;
+        max-height: 245px;
+        padding: 8px;
+    }
+`;
+
+export const Picture = styled.img`
+    max-width: 100%;
+    border-radius: ${({ theme }) => theme.borderRadius.small};
+`;
+
+export const NoPictureElement = styled.div`
```

Comment:
To nie element tylko container. Lepiej zrobić tak jak poprzednio - div z szarym tłem do którego wrzucasz zdjęcie lub ikonę

Reply:
Tutaj nie do końca zrozumiałem 🙅